### PR TITLE
Update strings from review.

### PR DESCRIPTION
**Pull request by pocmo**
Those are the new, reviewed strings from Michelle.

### CODE REVIEW

**Comment by pocmo** [comment]
Okay, lint doesn't like that we have translated strings now that are not used anymore... :)

**Comment by pocmo** [comment]
I disabled the lint checks for `MissingTranslation` and `ExtraTranslation`. With our translation process it is possible that we will have missing translations (just not translated yet) or extra translations (the next import should remove them) temporarily.

File: components/feature/sitepermissions/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -1,43 +1,33 @@
...
     <!-- Option in a dialog for requesting a camera permission, this option will give access to
          back facing camera-->
-    <string name="mozac_feature_sitepermissions_back_facing_camera">Back facing camera</string>
-
+    <string name="mozac_feature_sitepermissions_back_facing_camera2">Rear-facing camera</string>
     <!-- Option in a dialog for requesting a camera permission, this option will give access to
      front facing camera-->
-    <string name="mozac_feature_sitepermissions_selfie_camera">Selfie back camera</string>
-
+    <string name="mozac_feature_sitepermissions_selfie_camera2">Front-facing camera</string>
```
Comment (Amejia481) [inline components/feature/sitepermissions/src/main/res/values/strings.xml]: Some of the strings values from site permissions have been updated to more like Fennec style.  I took them from the Fenix mock-ups, now we are changing them, are we going to provide a way for apps to customize them or overridden the strings on the apps will be the preferred alternative?   As we are changing the strings ids the last alternative could be fragile.

Comment (pocmo) [reply components/feature/sitepermissions/src/main/res/values/strings.xml]: Those strings are superseding the ones from the mocks. Fenix is supposed to use those updated strings.

Comment (Amejia481) [reply components/feature/sitepermissions/src/main/res/values/strings.xml]: 👍

**Comment by pocmo** [comment]
Looks like I missed some tests that used those strings. I am going to update the PR.

**Pull request merged**